src/snowflake/snowpark/table.py

```diff
@@ -655,3 +661,11 @@ def merge(
             if block
             else result
         )
+
+    def drop_table(self):
+        """Drops the table from the Snowflake database.
+
+        Note that subsequent API such as ``select()``, ``collect()`` on this ``Table`` instance and the derived DataFrame will raise errors because the underlying
```

Comment:
```suggestion
        Note that subsequent operations such as :meth:`DataFrame.select`, :meth:`DataFrame.collect` on this ``Table`` instance and the derived DataFrame will raise errors because the underlying
```